.github/workflows/zeebe-ci.yml

```diff
@@ -85,7 +85,7 @@ jobs:
       - uses: ./.github/actions/build-zeebe
         id: build-zeebe
         with:
-          maven-extra-args: -T1C
+          maven-extra-args: -T1C -PskipFrontendBuild
```

Comment:
maybe we should update `./.github/actions/build-zeebe` to build only zeebe modules with `-pl zeebe -am -amd` arguments